content/en/serverless/azure_functions/_index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Tracing Azure Functions
```

Comment:
Maybe title should be `Install Serverless Monitoring for Azure Functions` to be in line with https://docs.datadoghq.com/serverless/aws_lambda/installation?